classes/options-general.php

```diff
@@ -591,31 +591,31 @@ class='howto'><?php _e( 'Select "No Limit" to display all results within the sea
 									<!-- Optional Features -->
 									<div class="postbox">
 
-										<h3><?php _e( 'Optional / Experimental Features', 'simplemap' ); ?></h3>
+										<h3><?php _e( 'Optional / Experimental Features', 'SimpleMap' ); ?></h3>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->